packages/fhir-converter/src/templates/cda/Header.hbs

```diff
@@ -1,74 +1,76 @@
-{{#with msg.ClinicalDocument}}
-    {{>Resources/Composition.hbs composition=this ID=(generateUUID (toJsonString this))}},
+{{#with msg.ClinicalDocument as |document|}}
```

Comment:
systematically removing this and .. from templating system. Very confusing and time and time again source of context related bugs